### PR TITLE
Update Build-a-Block-Editor.md

### PR DESCRIPTION
**Pull request by nathanwoulfe**
Hi team, noticed a few grammar/spelling/readability issues in the block editor docs, so have fixed 'em up. No changes to the intent of the content, just makes it a bit easier to read.

N

### CODE REVIEW

File: Extending/Property-Editors/Build-a-Block-Editor.md [hunk 1]
```diff
@@ -4,9 +4,7 @@ versionFrom: 8.7.0
 
 # Build a Block Editor
 
-To read this document you must make your self confident with the basics of developing a custom Property Editor for Umbraco.
-
-[Read more](https://our.umbraco.com/documentation/Extending/Property-Editors/).
+Before reading this document familiarise yourself with [the basics of developing a custom Property Editor for Umbraco](https://our.umbraco.com/documentation/Extending/Property-Editors/).
```
Comment (sofietoft) [inline Extending/Property-Editors/Build-a-Block-Editor.md:7]:
```suggestion
Before reading this document we highly recommend that you familiarise yourself with [the basics of developing a custom Property Editor for Umbraco](https://our.umbraco.com/documentation/Extending/Property-Editors/).
```

File: Extending/Property-Editors/Build-a-Block-Editor.md [hunk 2]
```diff
@@ -98,17 +96,17 @@ In the following example the layout object "MyOwn.UnicornBlocksEditor" is of typ
...
+To help your Block Editor manage the Data Structure presented above, we have made a Model Object called BlockEditorModelObject which helps manage the basic parts of a Block Editor.
 
 To get a better understanding of what the Model Object does for you, we need to look at some usages of the Model Object.
 
 ### Maintain and work with the Layout of a Block Editor
 
-The `layout` of a Block Editor can be of any structure. Therefor the Model Object(BlockEditorModelObject) cannot maintain this data. This makes our usage of Model Object a bit more complex as we offen will be giving the Model Object a reference to an entry of the `layout` for then to performe an action and if what went well we might have to reflect this change back to the `layout`.
+The `layout` of a Block Editor can be any structure. Therefore the Model Object (BlockEditorModelObject) cannot maintain this data. This makes our usage of Model Object a bit more complex as we often will be giving the Model Object a reference to an entry of the `layout` to then perform an action, which may then need to reflect changes back to the `layout`.
```
Comment (sofietoft) [inline Extending/Property-Editors/Build-a-Block-Editor.md:105]: Small suggestion, to shorten the last sentence just a bit.
Let me know what you think!

```suggestion
The `layout` of a Block Editor can be any structure. Therefore the Model Object (BlockEditorModelObject) cannot maintain this data. This makes our usage of Model Object a bit more complex as we often will be giving the Model Object a reference to an entry of the `layout` to then perform an action. This action may then need to reflect changes back to the `layout`.
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
Great suggestions @nathanwoulfe ! 💪 
Really makes the article flow a lot better.

I've made a couple of suggestions here and there.
Take a look, once you have a few minutes.

**Comment by nathanwoulfe** [comment]
Thanks @sofietoft, completely agree with your suggestions 👍

**Review by sofietoft — APPROVED** [review]
Thanks for applying the suggestions @nathanwoulfe !

Let's get this merged then 😁 
Cheers!

**Pull request merged**